tests/firsttests.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from '@playwright/test';
+
+test('Test 1: Verify login with valid credentials', async ({ page }) => {
+    await page.goto(process.env.WEB_URL + '/auth/login');
```

Comment:
Since we added url to playwright config, we can use relative path here:
```suggestion
    await page.goto('/auth/login');
```